src/components/ContinuationIn/SummaryBusinessHomeJurisdiction.vue

```diff
@@ -355,4 +375,18 @@ article:not(:first-child) {
 article:not(:last-child) {
   padding-bottom: 0;
 }
+
+// align the list v-icons and text
+.continuation-authorization-summary {
+  ul {
+    list-style: none;
+    margin-left: 2rem;
+    padding-left: 0;
+
+    li > i {
+      margin-left: -2rem;
+      padding-right: 10px;
+    }
+  }
+}
```

Comment:
This is to line the icon with the column, have proper whitespace, and indent the text block, ie:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/d6aeb9fd-f098-4bd1-80c9-c4b24b907bd2)